src/quest_manager/templates/quest_manager/submission_buttons_form_student.html

```diff
@@ -0,0 +1,17 @@
+<div class="btn-group" role="group">
+
+    {% if not completed %}
+        <button type='submit' name='complete' class='btn btn-info'
+                title='RE-SUBMIT this quest for completion'>
+        <i class="fa fa-fw fa-check"></i>
+        Re-Submit Quest
+        </button>
+    {% else %}
+        <button type='submit' name='comment' class='btn btn-primary'
+                title='COMMENT on this submission'>
```

Comment:
Change title to "Add a comment to this submission" (and change the old one too if there is something similar elsewhere)

Reply:
button title changed.